autogluon_zeroshot/metrics/_roc_auc_cpp/__init__.py

```diff
@@ -47,18 +47,22 @@ def _compile(self):
         # execute compilation command
         print(f"Running \"{compile_command}\" to compile c++ auc implementation.")
         with open("std.out", "w") as stdout:
-            proc = subprocess.Popen(compile_command.split(" "), shell=False, stdout=stdout, cwd=Path(__file__).parent)
+            with open("std.err", "w") as stderr:
```

Comment:
(minor) why add to stderr?

Reply:
So that you can check the error produced by the compilation, otherwise it is not available.